GANDLF/models/lightning_module.py

```diff
@@ -1,22 +1,53 @@
+import os
+import time
+import psutil
+import torch
+import torchio
+import warnings
+from medcam import medcam
+from copy import deepcopy
+from statistics import mean
 import lightning.pytorch as pl
+from lightning.pytorch.utilities import rank_zero_only
+from lightning.pytorch.callbacks import EarlyStopping
+from GANDLF.logger import Logger
 from GANDLF.models import get_model
 from GANDLF.optimizers import get_optimizer
 from GANDLF.schedulers import get_scheduler
 from GANDLF.losses.loss_calculators import LossCalculatorFactory
 from GANDLF.metrics.metric_calculators import MetricCalculatorFactory
 from GANDLF.utils.pred_target_processors import PredictionTargetProcessorFactory
 
-from copy import deepcopy
+from GANDLF.utils import (
+    optimize_and_save_model,
+    write_training_patches,
+    one_hot,
+    print_model_summary,
+    get_date_time,
+    save_model,
+    load_model,
+    version_check,
+    BEST_MODEL_PATH_END,
+    INITIAL_MODEL_PATH_END,
+    LATEST_MODEL_PATH_END,
+)
+
+from overrides import override
+from typing import Tuple, Union, Dict, List
 
 
 class GandlfLightningModule(pl.LightningModule):
-    def __init__(self, params: dict):
+    def __init__(self, params: dict, output_dir: str):
         super().__init__()
+        self.output_dir = output_dir
         self.params = deepcopy(params)
+        self.current_best_loss = 1e7
```

Comment:
Does this make sense? 

```suggestion
        self.current_best_loss = sys.float_info.max
```

Reply:
totally, forgot about that!